vaadin-tabs-flow-parent/vaadin-tabs-flow/src/main/java/com/vaadin/flow/component/tabs/TabSheetVariant.java

```diff
@@ -22,6 +22,10 @@
  */
 public enum TabSheetVariant implements ThemeVariant {
     //@formatter:off
+    /**
+     * @deprecated Use {@code TabVariant.LUMO_ICON_ON_TOP} instead.
```

Comment:
This is potentially confusing as it could be interpreted to use `TabVariant` together with `TabSheet` or `Tabs`, which doesn't work. Let's add that it should be used on `Tab` instances instead.

Reply:
Thanks, done.